pkg/apis/aws/v1alpha1/account_types.go

```diff
@@ -112,6 +114,7 @@ const (
 // +kubebuilder:printcolumn:name="Claimed",type="boolean",JSONPath=".status.claimed",description="True if the account has been claimed"
 // +kubebuilder:printcolumn:name="Claim",type="string",JSONPath=".spec.claimLink",description="Link to the account claim CR"
 // +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp",description="Age since the account was created"
+// +kubebuilder:resource:path=accounts,scope=Cluster
```

Comment:
Yeah, this is where the Cluster scoping came from -- why is this delta here?

Reply:
That is probably a misunderstanding on my part. The way the documentation was worded, I thought it needed to be cluster scoped since it was working within it's own namespace, as well as the UHC namespaces for each account.

Thanks for the catch!

Reply:
Mm, I think I see where the confusion came from. The operator itself needs to be cluster-scoped so it can watch & do things across namespaces as you say. But the CRs themselves are namespace-scoped.